src/Test.UnitTests.Sarif.Driver/Test.UnitTests.Sarif.Driver.csproj

```diff
@@ -25,7 +25,7 @@
 
   <Target Name="CopyFunctionalTestData" AfterTargets="Build">
     <ItemGroup>
-      <TestFiles Include="$(SolutionDir)FunctionalTestData\**\*" />
+      <TestFiles Include="..\FunctionalTestData\**\*" />
```

Comment:
>FunctionalTestData [](http://example.com/codeflow?start=29&length=18)

A more advanced change would be to use a special property that is $(SolutionDir) by default but which is overwritten when this repo is included as a submodule.

We can take this improved change later, if there's a need.